### PR TITLE
Add cloudevents proposal

### PR DESCRIPTION
**Pull request by ruromero**
Addresses  #10 

Signed-off-by: ruromero <rromerom@redhat.com>

### CODE REVIEW

File: proposals/cloudevents.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+# Proposal: CloudEvents as unified event data format
```
Comment (ilan-pinto) [inline proposals/cloudevents.md:1]: general comments: 
- suggesting to add the software application that already supports cloud events.  
- should we mention as well CloudEvents schema registry and service discovery?

**Review by ilan-pinto — COMMENTED** [review]
looks really good!!

File: proposals/cloudevents.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+[CloudEvents](https://github.com/cloudevents/spec) is a CNCF specification for describing event data in common formats to provide interoperability across services, platforms and systems.
+
+Mapping each incoming and outgoing message for each RESTful operation defined in the BIAN Semantic APIs will allow to decouple the protocol from the message enabling any type of communication pattern.
+
+Moreover by using an agreed CloudEvent structure as the payload, the conversion between different transports will become much easier.
+
+![cloudevents payload](./images/cloudevents-payload.png)
+
+### Mapping a RESTful operation to a CloudEvent
```
[the remark this answers is not in the document]
Comment (ruromero) [reply proposals/cloudevents.md:67]: This is specific to the HTTP transport. The content-type and http headers are described already in the Cloudevents spec for each independent transport

[the remark this answers is not in the document]
Comment (ilan-pinto) [reply proposals/cloudevents.md:67]: where?

[the remark this answers is not in the document]
Comment (ruromero) [reply proposals/cloudevents.md:67]: For HTTP: https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#313-metadata-headers
For Kafka: https://github.com/cloudevents/spec/blob/v1.0.1/kafka-protocol-binding.md#323-metadata-headers

**Pull request merged**